src/reducers/GlobalReducer.ts

```diff
@@ -6,6 +6,6 @@ export default function(state = {}, action: { payload: any; type: string }) {
     case ROOT_KEY_CHANGED:
       return { ...state, rootElementKey: Utils.generateUuidV4() };
     default:
-      return state;
+      return { ...state, isMobile: Utils.getResponsive() };
```

Comment:
What `action.type` triggers this case? Any reason why it doesn't have an explicit type?

Reply:
I made it by default to start working with this value in the state. But it should be reactive and having its own type yes. I planned to do it in the future commits. I should have commented it to explain the why :)